assets/js/atomic/blocks/product-elements/product-image-gallery/edit.tsx

```diff
@@ -58,7 +58,7 @@ const Edit = ( { context }: Props ) => {
 		);
 	}
 	// We have work on this case when we will work on the Single Product block.
```

Comment:
Should this comment be removed, now? Or what's missing here?

Reply:
Luigi was working on this component, I think he mentioned that this comment was there because in the future we could think about displaying the image for the selected product instead of the image placeholder; perhaps we should create an issue and remove the comment? 🤔

Reply:
cc'ing @gigitux for his thoughts on this. :point_up:

Reply:
At least, for now, we decided to use the `Product Image` block for the Single Product Block. In the future, we should add support for the `Product Image Gallery`. (we don't have a dedicated roadmap) - related discussion ( p1678382112656919-slack-C02FL3X7KR6).
It makes sense to open an issue and link it in this comment. What do you think?